app/controllers/case_workers/admin/management_information_controller.rb

```diff
@@ -14,14 +16,13 @@ def index
   def download
     log_download_start
     record = Stats::StatsReport.most_recent_by_type(params[:report_type])
-    if record.document?
-      data = open(record.document_url).read
-      content_type = record.document_content_type
+
+    if record.document.attached?
+      # TODO: Change service_url to url when moving to Rails 6.1
+      redirect_to record.document.blob.service_url(disposition: 'attachment')
```

Comment:
are we not using an [expiring url like this perhaps](https://api.rubyonrails.org/v6.1.3/classes/ActiveStorage/Blob.html#method-i-service_url)

Reply:
URLs have a 5 minute expiry by default but perhaps it should be explicitly?